src/hyperspheres.jl

```diff
@@ -88,3 +57,14 @@ function create_bsphere(m::Metric,
 
     return HyperSphere(SVector{N,T}(center), rad)
 end
+
+@inline function interpolate(::M, c1::V, c2::V, x, d) where {V <: AbstractVector, M <: NormMetric}
```

Comment:
Why move this function?

Reply:
I had two versions locally, the previous one, and this one without the array buffer variable `ab`. It turns out that in the sequential code, the compiler is able to get rid of the allocations without explicitly pre-allocating an `ArrayBuffer` variable. In the parallel code, having an array buffer leads to race conditions, which is why I wrote this modification. 

I can move it back to where it was in the file.